src/main/flight/pid.h

```diff
@@ -115,6 +115,7 @@ typedef struct pidProfile_s {
     uint8_t throttle_boost;                 // how much should throttle be boosted during transient changes 0-100, 100 adds 10x hpf filtered throttle
     uint8_t throttle_boost_cutoff;          // Which cutoff frequency to use for throttle boost. higher cutoffs keep the boost on for shorter. Specified in hz.
     uint8_t  iterm_rotation;                    // rotates iterm to translate world errors to local coordinate system
+    uint8_t  smart_feedforward;
```

Comment:
Maybe some description here would be great and better fitting with the others

Reply:
@joelucid: Are you going to add a description here?

Reply:
Done